src/FinchNotation/syntax.jl

```diff
@@ -105,7 +105,7 @@ end
 function (ctx::FinchParserVisitor)(ex::Symbol)
     if ex == :_ || ex == :(:)
         return :($dimless)
-    elseif ex in evaluable_exprs
+    elseif ex in evaluable_exprs || ex == :Eps 
```

Comment:
Could you just export the symbol Eps? Or perhaps add Eps to evaluable exprs?

Reply:
After you do that, feel free to push the automerge button and github will merge it after tests pass